libraries/app/include/scorum/app/scorum_api_objects.hpp

```diff
@@ -589,6 +592,34 @@ struct atomicswap_contract_result_api_obj
     }
 };
 
+struct registration_committee_api_obj
+{
+    registration_committee_api_obj()
+    {
+    }
+
+    registration_committee_api_obj(const registration_pool_object& reg_committee)
+        : invite_quorum(reg_committee.invite_quorum)
+        , dropout_quorum(reg_committee.dropout_quorum)
+        , change_quorum(reg_committee.change_quorum)
+    {
+    }
+
+    protocol::percent_type invite_quorum = 0u;
+    protocol::percent_type dropout_quorum = 0u;
+    protocol::percent_type change_quorum = 0u;
+};
+
+struct development_committee_api_obj : public api_obj<scorum::chain::dev_committee_object>
```

Comment:
I suggest, that it can be replaced by: 

using development_committee_api_obj  = api_obj<scorum::chain::dev_committee_object>;